README.md

```diff
@@ -80,6 +80,49 @@ data_importer.wait_until_complete
  => Success()
 data_importer.instance_hrid
  => Success("in00000000010")
+
+# Get list of organizations (filtered with an optional query)
+# see https://s3.amazonaws.com/foliodocs/api/mod-organizations/p/organizations.html#organizations_organizations_get
+ client.organizations
+ => {"organizations"=>[
+     {"id"=>"4b1a42f9-b310-492c-a71d-b8edcd30ac0c",
+    "name"=>"Seventh Art Releasing",
+    "code"=>"7ART-SUL",
+    "exportToAccounting"=>true,
+    "status"=>"Active",
+    "organizationTypes"=>[],
+    "aliases"=>[],
+    "addresses"=>],.....
+    "totalRecords"=>100}
+
+client.organizations(query: 'name="Seventh"')
+=> {"organizations"=>[....
+
+# Get list of organization interface items (filtered with an optional query)
+# see https://s3.amazonaws.com/foliodocs/api/mod-organizations-storage/p/interface.html#organizations_storage_interfaces_get
+client.organization_interfaces
+ => {"interfaces"=>
+        [{"id"=>"c6f7470e-6229-45ce-b3f9-32006e9affcf",
+          "name"=>"tes",
+          "type"=>["Invoices"],
+    .....],....
+    "totalRecords"=>100}
+
+ client.organization_interfaces(query: 'name="tes"')
```

Comment:
I assume that a query could also use the ID, correct? Like `query: 'id="some-uuid-of-an-interface"'`?

Reply:
no, you just need to do `GET /organizations-storage/interfaces/{id}`

Reply:
Yes, it will take any query that is supported by the API (it just passes it along without parsing).

```
result = FolioClient.organization_interfaces(query: 'id="c6f7470e-6229-45ce-b3f9-32006e9affcf"')
result
 =>
{"interfaces"=>
  [{"id"=>"c6f7470e-6229-45ce-b3f9-32006e9affcf",
    "name"=>"tes",
    "type"=>["Invoices"],
    "metadata"=>
     {"createdDate"=>"2023-02-16T22:27:51.515+00:00",
      "createdByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6",
      "updatedDate"=>"2023-02-16T22:27:51.515+00:00",
      "updatedByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6"}}],
 "totalRecords"=>1}
```

This is basically the same thing as using the interface details query:

```
result = FolioClient.interface_details(id: 'c6f7470e-6229-45ce-b3f9-32006e9affcf')
result
 =>
{"id"=>"c6f7470e-6229-45ce-b3f9-32006e9affcf",
 "name"=>"tes",
 "type"=>["Invoices"],
 "metadata"=>
  {"createdDate"=>"2023-02-16T22:27:51.515+00:00",
   "createdByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6",
   "updatedDate"=>"2023-02-16T22:27:51.515+00:00",
   "updatedByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6"}}
```

Reply:
I mean, you _could_ do `/organizations-storage/interfaces?query=id==c6f7470e-6229-45ce-b3f9-32006e9affcf` if you wanted to...

Reply:
If you don't pass any query to the `organization_interfaces` it will return everything.

```
result = FolioClient.organization_interfaces
result
{"interfaces"=>
  [{"id"=>"c6f7470e-6229-45ce-b3f9-32006e9affcf",
    "name"=>"tes",
    "type"=>["Invoices"],
    "metadata"=>
     {"createdDate"=>"2023-02-16T22:27:51.515+00:00",
      "createdByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6",
      "updatedDate"=>"2023-02-16T22:27:51.515+00:00",
      "updatedByUserId"=>"38524916-598d-4edf-a2ef-04bba7e78ad6"}},
   {"id"=>"65d30c15-a560-4064-be92-f90e38eeb351",
    "name"=>"GOBI-FTP",
    "uri"=>"ftp://ftp.ybp.com",
    "notes"=>"THIS IS AN EXAMPLE - IF ANY PROBLEMS ARRISE, DELETE",
    "type"=>["Other"],
    "metadata"=>
     {"createdDate"=>"2023-03-09T22:30:53.993+00:00",
      "createdByUserId"=>"ef452e80-116b-4e6a-b820-3827a3c87c17",
      "updatedDate"=>"2023-03-09T22:30:53.993+00:00",
      "updatedByUserId"=>"ef452e80-116b-4e6a-b820-3827a3c87c17"}},
   {"id"=>"3a00800c-c6a9-447d-96b5-cf5786d69df3",
    "name"=>"Test Interface",
    "type"=>["Other"],
    "metadata"=>
     {"createdDate"=>"2023-03-27T20:16:45.467+00:00",
      "createdByUserId"=>"950344dc-af40-58e1-aa90-c015b7ac51f2",
      "updatedDate"=>"2023-03-27T20:16:45.467+00:00",
      "updatedByUserId"=>"950344dc-af40-58e1-aa90-c015b7ac51f2"}}],
 "totalRecords"=>3}
```

```

Reply:
Oh, I see it now, that's the interface_details method. We may also need to add a method for the `/organizations-storage/interfaces/{id}/credentials` endpoint, but I want to chat with @jermnelson about the API requirements first, so that could easily be a follow up PR.